src/main/java/seedu/address/model/person/Telegram.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's Telegram username in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTele(String)}
+ */
+public class Telegram {
+
+    public static final String MESSAGE_CONSTRAINTS = "Telegram usernames can take any values, "
```

Comment:
I feel like maybe the constraint should be that it begins with an @, not sure about this though we can discuss tomorrow